docs/Basic-Guide.md

```diff
@@ -74,18 +77,18 @@ More information and documentation is provided in the
 
 ## Training the Brain with Reinforcement Learning
 
-### Setting the Brain to External
+### Adding a Brain to the training session
 
-Since we are going to build this environment to conduct training, we need to set
-the Brain used by the Agents to **External**. This allows the Agents to
+Since we are going to build this environment to conduct training, we need to add 
+the Brain to the training session. This allows the Agents to
 communicate with the external training process when making their decisions.
 
-1. In the **Scene** window, click the triangle icon next to the Ball3DAcademy
-   object.
-2. Select its child object **Ball3DBrain**.
-3. In the Inspector window, set **Brain Type** to **External**.
+1. Give to the agents you want to train the **Ball3DBrain**. 
```

Comment:
This is worded a little strangely. Perhaps "Assign the **Ball3DBrain** to the agents you would like to train.